### PR TITLE
Test rollback document suggested modification

### PR DESCRIPTION
**Pull request by wxw310415**

### CODE REVIEW

File: src/en/guide/testing/integrationTesting.adoc [hunk 1]
```diff
@@ -36,7 +36,8 @@ class ExampleSpec extends Specification {
 
 The `Rollback` annotation ensures that each test method runs in a transaction that is rolled back. Generally this is desirable because you do not want your tests depending on order or application state.
 
-In Grails 3.0 tests rely on `grails.transaction.Rollback` annotation to bind the session in integration tests. Though each test method transaction is rolled back, the `setup()` method uses a separate transaction that is not rolled back.
+In Grails 3.0 tests rely on `grails.transaction.Rollback` annotation to bind the session in integration tests. Though each test method transaction is rolled back, the `setup()` method uses a separate transaction that is not rolled back and the `setup()` method will be executed automatically when the tests is executed.
```
Comment (niravassar) [inline src/en/guide/testing/integrationTesting.adoc]: I think this `... is not rolled back and the setup()...` should be broken into a sentence like this:

 `is not rolled back. The setup() `

File: src/en/guide/testing/integrationTesting.adoc [hunk 2]
```diff
@@ -72,6 +73,11 @@ import spock.lang.*
 @Integration
 @Rollback
 class BookSpec extends Specification {
+    
+    void setup() {
+        // Do not use this method
```
Comment (niravassar) [inline src/en/guide/testing/integrationTesting.adoc]: These two lines should not be commented out in the example, and the comment should be
`// This line will not roll back`

**Review by niravassar — CHANGES_REQUESTED** [review]
Let me know what you think of these changes and we can merge it as i like the idea

**Comment by jamesfredley** [comment]
closed due to age

**Pull request closed without merging**